### PR TITLE
content added pull request #3

### LINKED ISSUE #3: wanna content added
content added

### PR DESCRIPTION
**Pull request by AppServiceProvider**
description +
issues fixes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # pullTest
-* In publishing and graphic design, Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. Lorem ipsum may be used as a placeholder before final copy is availablecontect add secondBranch
\ No newline at end of file
+* In publishing and graphic design, Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. Lorem ipsum may be used as a placeholder before final copy is availablecontect add secondBranch
+* Fill your designs with placeholders in a click: text, images and even data! Generate 'Lorem ipsum' to fill your text layers,Fill any shape with the perfect
```
Comment (romanofficial) [inline README.md:3]: content are copyright

Comment (AppServiceProvider) [inline README.md:3]: content is copy right

Comment (composerjson) [reply README.md:3]: ok

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # pullTest
-* In publishing and graphic design, Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. Lorem ipsum may be used as a placeholder before final copy is availablecontect add secondBranch
```
Comment (codingwithroman) [inline README.md:2]: @romanofficial you should change also

File: README.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
 # pullTest
```
Comment (composerjson) [inline README.md:1]: ??

**Review by composerjson — COMMENTED** [review]
wait for review

**Pull request merged**